importexport/musicxml/exportxml.cpp

```diff
@@ -5813,6 +5817,12 @@ static void measureStyle(XmlWriter& xml, Attributes& attr, const Measure* const
 
 static bool commonAnnotations(ExportMusicXml* exp, const Element* e, int sstaff)
       {
+      if (!exp->canWrite(e)) {
```

Comment:
I wonder when this `canWrite()` came into master...

Reply:
Hmm, the former apparently #21362, seems I left it out in the backport #304, which seems to look entirely different?